src/Database/Migrator.php

```diff
@@ -242,6 +263,32 @@ public function resolve($path, $file)
         }
     }
 
+    /**
+     * Initialize the Flarum database from a schema dump.
+     *
+     * @param string $path to the directory containing the dump.
+     */
+    public function installFromSchema(string $path)
+    {
+        $schemaPath = "$path/install.dump";
+
+        $currDumpFile = file_get_contents($schemaPath);
+
+        file_put_contents($schemaPath, str_replace('db_prefix_', $this->tablePrefix, $currDumpFile));
```

Comment:
Is this writing over the original file?

Won't this fail if the user delete their `config.php`, the database and try to install again with a different prefix?

I see later in the code this might be reverted (line 289?) but what if the installation errors in between?

Wouldn't it be safer to create a second temporary file?